test/node/src/aggregate-function.test.ts

```diff
@@ -1106,6 +1107,56 @@ for (const dialect of DIALECTS) {
 
       await query.execute()
     })
+
+    describe(`should execute order-sensitive aggregate functions`, () => {
+      // @todo add test for mssql when implement `WITHIN GROUP ( order_by_clause )` clause
+      if (dialect !== 'mssql') {
```

Comment:
Now that we have many dialects, it's best to be explicit.

```suggestion
      if (dialect === 'postgres' || dialect === 'mysql' || dialect === 'sqlite') {
```